console/src/components/Services/ApiExplorer/ApiRequest/ApiRequest.js

```diff
@@ -613,8 +613,8 @@ class ApiRequest extends Component {
             claimData =
               claimFormat === 'stringified_json'
                 ? generateValidNameSpaceData(
-                    JSON.parse(payload[claimNameSpace])
-                  )
+                  JSON.parse(payload[claimNameSpace])
```

Comment:
@rikinsk this happened when I merged master. I'm not sure which one is correct but it looks more or less the same. Let me know if I should change this.

Reply:
@marionschleifer It is because of pre-commit hook which refactors the JS code. Use `git commit --no-verify` when you're merging master.